base/cvd/BUILD.mkbootimg.bazel

```diff
@@ -0,0 +1,14 @@
+package(
+    default_visibility = ["@//:android_cuttlefish"],
+)
+
+cc_library(
+    name = "bootimg_header",
+    hdrs = ["include/bootimg/bootimg.h"],
+    strip_include_prefix = "include/bootimg",
+)
```

Comment:
It doesn't look like anything depends on this `cc_library` yet, is that planning to come later?

Reply:
Yes, it'll be used by fastboot.

Reply:
See PR #1075
Draft so I can let CI & merge complete on this dependency in case it needs to be reverted.